docs/en/reference/sql/dql/LIMIT_CLAUSE.md

```diff
@@ -16,17 +16,18 @@ SELECT ... LIMIT ...
 ```
 
 ## Boundary Description
+For the standalone version, `LIMIT` is supported in all conditions. For the cluster version, the execution modes which support this clause is shown below.
```

Comment:
the execution modes which support this clause is shown below
the execution modes, which support this clause, are shown below

Reply:
Thank you for reminding me！